WordPress/build.gradle

```diff
@@ -467,7 +467,7 @@ dependencies {
         exclude group: "com.android.volley"
         exclude group: 'org.wordpress', module: 'utils'
     }
-    implementation (project(path:':libs:analytics:WordPressAnalytics')) {
+    implementation (project(path:':libs:analytics')) {
```

Comment:
Should we use `:libs:WordPressAnalytics` instead, so it's consistent with `:libs:WordPressProcessors`, `:libs:WordPressAnnotations` etc?

Reply:
👋 @oguzkocer and thank you so much for the suggestion! 🙏 

I actually considered that too while doing this change, but I ended up in favor of the plain `analytics` naming instead of `WordPressAnalytics` for the below two main reasons:
1. We are already in the context of the `WordPress` repo, as such, having to add the `WordPress` prefix to every module seems unnecessary.
2. This repo, in addition to the `WordPress` app is also building the `Jetpack` app. As such, it might be better to remove the word `WordPress` from every module so that to avoid any `Jetpack` related naming confusion.

To that end, I was thinking to also rename the `WordPressProcessors` and `WordPressAnnotations` modules to plain `processors` and `annotations` in order to have every module naming consistent from now on.

In the end, I am planning to end up with the below module names:
- analytics (from `WordPressAnalytics`)
- editor (from `WordPressEditor`)
- editor-example (from `editor/example`)
- image-editor (from `ImageEditor`)
- mocks (from `WordPressMocks`)
- networking (from `WordPressNetworking`)
- annotations (from `WordPressAnnotations`)
- processors (from `WordPressProcessors`)

Let me know what you think. 🤔 🙇 

PS: You will also notice the `editor-example`:
- I also want to have it build and be part of our active modules as it is currently not.
- I am thinking of moving it up to the same level as every other module for now, just for simplicity sake and until we figure out how modularization and the organization of it would look like as we progress forward with the main modularization project (TBD).

Reply:
Great, yes, consistency matters, thanks for giving me the 👍 on it @oguzkocer ! 🙇